tutorials/abap-custom-ui-bas-develop-s4hc/abap-custom-ui-bas-develop-s4hc.md

```diff
@@ -22,10 +22,10 @@ author_profile: https://github.com/Liebherr
   - How to deploy the UI as custom application to your SAP S/4HANA Cloud system
   - This process of custom UI development is the way to go if the UI generated within the SAP S/4HANA Cloud system does not match your needs
 
-## Intro
-You can get a generated UI for your custom business object and adapt it to your needs within the SAP S/4HANA Cloud system (in-app-extensibility). As building UIs in SAP BTP (side-by-side extensibility) offers more possibilities than in-app-extensibility, this tutorial gives you an idea how to build a UI based on a custom business object's OData service in SAP Business Application Studio, test it, and get it into the SAP S/4HANA Cloud system.
-
->Tutorial last updated with SAP S/4HANA Cloud Release 2302
+**Additional Info**
+> - This tutorial gives you an idea how to build a UI with all SAPUI5 options in SAP Business Application Studio, test it, and get it into the SAP S/4HANA Cloud system. If you only require a generated UI and maybe adapting it with restricted options within the SAP S/4HANA Cloud system (in-app-extensibility) check out [Generate the UI for a Custom Business Object and grant Access](abap-extensibility-cbo-ui-generation) and [Adapt the UI of a Business Object](abap-extensibility-cbo-ui-adaptation)
```

Comment:
I would argue that if the tutorial shows all the needed steps to build a UI, test it, and get it into the system, you should rephrase this sentence: do not say 'This tutorial gives you an idea' but 'This tutorial shows/illustrates all the needed steps to.. etc' or something similar